### DISCUSSION TITLE
Week 6: July 22-26

### DISCUSSION THREAD

**Discussion by eeholmes** [SAFS-Varanasi-Internship/Summer-2024#20] [General]
For Yifei:

1) look at performance for years besides 2022.
2) 4-panel with 4 different years
3) Next predictions for the whole area. Do a comparison to CHL_cmes-gapfree
4) Start a tutorial on UNet.
5) Start putting together a short presentation on your results.
<img width="481" alt="image" src="https://github.com/user-attachments/assets/568ba5d2-32e7-417c-afd6-60d1a6766f0f">

**Comment by eeholmes** [comment]
@yifeihang add more figures and progress here for what you showed.

**Comment by eeholmes** [comment]
<img width="579" alt="image" src="https://github.com/user-attachments/assets/3f9d40b8-9421-44a3-ab7c-10530a06cc86">

**Comment by eeholmes** [comment]
@yifeihang and @ShridharS19 Can you add some pictures of your results that you showed on Tuesday? Just paste in here. Thanks!

**Reply by ShridharS19** [reply]
![Uploading notebook result.png…]()

**Comment by yifeihang** [comment]
![2days_model_comparisons](https://github.com/user-attachments/assets/ef40d6e1-5860-4215-bce4-b1faad645d72)
![DoubleConv_2days_pred](https://github.com/user-attachments/assets/1ce29a7c-0c3d-4ba5-a0ae-b58fc6a16eff)

**Comment by yifeihang** [comment]
07/25/24 (Thurs.)
(Note: in all line plots, loss = MAE)
- With model loss function = mae:
Loss comparison plots and prediction with double conv:
![DoubleConv_2days_pred_20200813](https://github.com/user-attachments/assets/1f08d838-1dfa-4e46-9525-14a52092f789)
![2days_model_comparisons_2014_20_21_22](https://github.com/user-attachments/assets/3bd74c2a-a90d-48f8-8459-dbac2bffc5b6)
predicting the whole area:
![2days_model_comparisons_2014_20_21_22_full_predictions](https://github.com/user-attachments/assets/579c7097-4348-4556-9185-bc520ab7c29f)
![DoubleConv_2days_pred_full_20200913](https://github.com/user-attachments/assets/2208922f-f7b8-42b6-a650-6843ff6d3403)

- With model loss function = mse:
![MSE_DoubleConv_2days_pred_20200813](https://github.com/user-attachments/assets/c3cabe12-0456-4f92-a6d7-efa190b42508)
![MSE_2days_model_comparisons_2014_20_21_22](https://github.com/user-attachments/assets/5dcabcba-bcf9-42c9-bb6d-0ccbdc933cf5)
Full area prediction:
![MSE_DoubleConv_2days_pred_full_20200913](https://github.com/user-attachments/assets/ed7e7a10-a973-401c-9da0-7afb38399a10)
![MSE_2days_model_comparisons_2014_20_21_22_full_predictions](https://github.com/user-attachments/assets/0e37b940-c8f0-4646-be11-dd902e898296)

Analysis:
- Using MSE for training seem to outperform MAE (lower MAE during evaluation)
- DoubleConv outperforms other models in nearly all circumstances (except for full area prediction with loss=mae)
- There does exist a seasonal trend, with May and June being overall the easiest to predict, and August (and February + March for some models) being the most difficult

**Reply by eeholmes** [reply]
@yifeihang Great results!

**Comment by eeholmes** [comment]
@yifeihang @ShridharS19 I think you 2 should submit an abstract to this. I will cover the cost: $29 for a student membership (1 person) and $40 for a student abstract submission. The conference is 9-13 December 2024 in Wash DC; most people go for 3-4 days. This unfortunately overlaps with finals but even if one of you were not able to attend, I could present the poster. Not ideal but possible. If the abstract were accepted (it's competitive), it is a great thing to have on a graduate school application and good experience to create a poster for a conference.

Here are example of the abstracts from last year to the same undergraduate poster session:
https://agu.confex.com/agu/fm23/meetingapp.cgi/Session/186506

Deadline for abstract submission is July 31 end of day

**Reply by ShridharS19** [reply]
That sounds exciting! I will do some more work on the weekend and consult with yifeis repo to get more MAE plots and we can talk on submitting on Monday?

**Comment by ShridharS19** [comment]
@yifeihang @eeholmes https://www.overleaf.com/5885443374bpnwhccwhkwq#0e0cea Here is an overleaf template I drew up with GPT. Me and Yifei will work on it during our meeting at 4pm today.

**Reply by yifeihang** [reply]
Can you share the link to our meeting?
